src/curated-corpus/components/ScheduleItemForm/ScheduleItemForm.test.tsx

```diff
@@ -66,4 +66,84 @@ describe('The ScheduleItemForm component', () => {
     // "Save" and "Cancel" buttons.
     expect(buttons).toHaveLength(2);
   });
+
+  it('does not pre-select a scheduled surface if none was passed in and the user has access to many', () => {
+    render(
+      <MuiPickersUtilsProvider utils={LuxonUtils}>
+        <ScheduleItemForm
+          data-testId="surface-selector"
+          handleDateChange={jest.fn()}
+          lookupCopy=""
+          selectedDate={DateTime.local()}
+          onSubmit={handleSubmit}
+          scheduledSurfaces={scheduledSurfaces}
+          approvedItemExternalId={'123abc'}
+        />
+      </MuiPickersUtilsProvider>
+    );
+
+    const select = screen.getByLabelText(
+      'Choose a Scheduled Surface'
+    ) as HTMLSelectElement;
+
+    // there should be an empty option and a single scheduled surface option
+    expect(select.options.length).toEqual(scheduledSurfaces.length + 1);
+
+    // the empty option should be selected
+    expect(select.options[0].selected).toBeTruthy();
+  });
+
+  it('pre-selects the passed in scheduled surface', () => {
+    render(
+      <MuiPickersUtilsProvider utils={LuxonUtils}>
+        <ScheduleItemForm
+          data-testId="surface-selector"
+          handleDateChange={jest.fn()}
+          lookupCopy=""
+          selectedDate={DateTime.local()}
+          onSubmit={handleSubmit}
+          scheduledSurfaces={scheduledSurfaces}
+          scheduledSurfaceGuid="NEW_TAB_EN_US"
+          approvedItemExternalId={'123abc'}
+        />
+      </MuiPickersUtilsProvider>
```

Comment:
@jpetto , I'm curious why you add `data-testId` to the component in the new tests but then don't query the DOM by this prop?

Reply:
oh! yeah, that `data-testId` isn't needed. at one point i was trying to find the react equivalent of `getElementById` and thought that might do it. (spoiler: in this case, it doesn't.) i just forgot to remove that attribute when i moved to `getByLabelText`.

Reply:
but now that i think about it, maybe `data-testId` would work - at that point i also didn't realize i had to cast the result `as HTMLSelectElement`.